ballerina/modules/composer/tests/schema_test.bal

```diff
@@ -35,7 +35,8 @@ function constructRGB(json data) returns json|schema:SchemaError {
 
 @test:Config {
     dataProvider: jsonLineDataGen,
-    groups: ["composer"]
+    groups: ["composer"],
+    enable: false
```

Comment:
Why did we disable this?

Reply:
This is due to a bug in the implementation. Will enable it once it is fixed. Created issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/4480

Reply:
Enabled in [3ec2028](https://github.com/ballerina-platform/module-ballerina-yaml/pull/9/commits/3ec20285a3af4c4280566410f3b825814c2d6c78)